### PR TITLE
Align CTAs for better & deep analytics

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
fixes [MKWEB-105 : Align CTA naming](https://linear.app/gitpod/issue/MKWEB-105/align-cta-naming)

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3858"></a>

### CODE REVIEW

File: src/lib/components/security/report.svelte [hunk 1]
```diff
@@ -13,7 +13,7 @@
 			community.
 		</p>
 		<LinkButton variant="secondary" size="medium" href="/contact/support"
-			>Report security concern</LinkButton
+			>Contact support</LinkButton
```
Comment (chrifro) [inline src/lib/components/security/report.svelte:16]: Can we keep this as it was and pre-select the subject "security?

File: src/lib/contents/pricing.ts [hunk 2]
```diff
@@ -35,7 +35,7 @@ export const pricingPlans: Pricing[] = [
 			'zero platform maintenance or overhead',
 			`multi-tenant set-up to minimize cloud costs`,
 		],
-		btnText: 'Get started',
+		btnText: 'Start for free',
```
Comment (chrifro) [inline src/lib/contents/pricing.ts:38]: keep old labels to differentiate between the free plan and "paid" plan.

File: src/lib/contents/pricing.ts [hunk 3]
```diff
@@ -112,7 +112,7 @@ export const empowermentFeatures: Card[] = [
 export const starterPlanTableData: FeatureTableColumn = {
 	link: {
 		href: 'https://gitpod.io/login',
-		label: 'Sign up',
+		label: 'Start for free',
```
Comment (chrifro) [inline src/lib/contents/pricing.ts:115]: This is not visible as we don't show the feature table, right?

**Review by chrifro — APPROVED** [review]
Awesome, thanks ✨

File: src/lib/contents/pricing.ts (hunk 3 above)
Comment (Siddhant-K-code) [reply src/lib/contents/pricing.ts:115]: yes 🙈

File: src/lib/contents/pricing.ts (hunk 2 above)
Comment (Siddhant-K-code) [reply src/lib/contents/pricing.ts]: fixed in [`70e4dc4`](https://github.com/gitpod-io/website/pull/3858/commits/70e4dc482b4e771a7d136dae9430872952eac816)

File: src/lib/components/security/report.svelte (hunk 1 above)
Comment (Siddhant-K-code) [reply src/lib/components/security/report.svelte]: fixed in [`70e4dc4`](https://github.com/gitpod-io/website/pull/3858/commits/70e4dc482b4e771a7d136dae9430872952eac816)

**Pull request merged**